cocoa/NSPasteboard.go

```diff
@@ -98,10 +99,51 @@ func (pb NSPasteboard) ClearContents() {
 // SetStringForType sets the given string as the representation for the specified type for the first item on the receiver.
 // https://developer.apple.com/documentation/appkit/nspasteboard/1528225-setstring?language=objc
 func (pb NSPasteboard) SetStringForType(s string, t NSPasteboardType) {
-	pb.Send("setString:forType:", core.NSString_FromString(s),  core.NSString_FromString(t))
+	pb.Send("setString:forType:", core.NSString_FromString(s),  core.NSString_FromString(string(t)))
 }
+
 // StringForType returns a concatenation of the strings for the specified type from all the items in the receiver that contain the type.
 // https://developer.apple.com/documentation/appkit/nspasteboard/1533566-stringfortype?language=objc
 func (pb NSPasteboard) StringForType(t NSPasteboardType) string {
-	return pb.Send("stringForType:",  core.NSString_FromString(t)).String()
+	return pb.Send("stringForType:",  core.NSString_FromString(string(t))).String()
+}
+
+// DataForType returns the data for the specified type from the first item in the receiver that contains the type.
+// https://developer.apple.com/documentation/appkit/nspasteboard/1531810-datafortype?language=objc
+func (pb NSPasteboard) DataForType(t NSPasteboardType) core.NSData {
+	return core.NSData{
+		Object: pb.Send("dataForType:",  core.NSString_FromString(string(t))),
+	}
+}
+
+// Types is an array of the receiver’s supported data types.
+// https://developer.apple.com/documentation/appkit/nspasteboard/1529599-types?language=objc
+func (pb NSPasteboard) Types() []NSPasteboardType {
+
+	o := pb.Get("types")
+	log.Println(o.Pointer())
+	arr := core.NSArray{o}
+	count := int(arr.Count())
+	types := make([]NSPasteboardType, count)
+	for i := 0; i < count; i++ {
```

Comment:
lets try adding a `Strings()` method to `NSArray` and have this just convert from `[]string` to `[]NSPasteboardType`